.github/workflows/ci-alpha.yml

```diff
@@ -49,3 +50,46 @@ jobs:
           buildargs: CUSTOM_REGISTRY_URL,CUSTOM_REGISTRY_TOKEN,WEB_SDK_DIST_TAG,UX_KIT_DIST_TAG
           buildoptions: "--pull"
           tags: "${{ env.WEB_SDK_DIST_TAG }}"
+
+  build_frontend_alpha_without_docker:
```

Comment:
maybe `build_frontend_alpha_static_content` would be more intuitive and descriptive?
the other job could be named `build_frontend_alpha_docker_image`, what do you think?